tests/golden.xml

```diff
@@ -1,61 +1,76 @@
+      id="0"
+      name="Some section"
+      package="Some section"
+      skipped="0"
+      tests="2"
+      time="0.000018308"
+      timestamp="2023-09-12T11:08:18.360279095Z">
+        <properties/>
+        <testcase
+          classname="Some section"
+          file="tests/ExampleSpec.hs"
+          line="19"
+          name="has a failure"
+          time="0.000012270">
+            <failure type="error">tests/ExampleSpec.hs:19:7
 
-expected: &#34;False&#34;
- but got: &#34;True&#34;
+expected: "False"
+ but got: "True"
 </failure>
-    </testcase>
-    <testcase
-      file="tests/ExampleSpec.hs"
-      line="15"
-      name="has an expectation"
-      classname="Some section"
-      time="0.000016818"/>
-  </testsuite>
-  <testsuite
-    name="Some section/A grouped context"
-    package="Some section/A grouped context"
-    id="1"
-    time="0.000021009"
-    timestamp="2021-05-17T15:36:10.205608409Z"
-    hostname="localhost"
-    tests="3" failures="0" errors="0" skipped="1">
-    <properties/>
-    <testcase
-      file="tests/ExampleSpec.hs"
-      line="26"
-      name="gets skipped"
-      classname="Some section/A grouped context"
-      time="0.000006765">
-      <skipped>tests/ExampleSpec.hs:26:9
+        </testcase>
+        <testcase
+          classname="Some section"
+          file="tests/ExampleSpec.hs"
+          line="16"
+          name="has an expectation"
+          time="0.000006038"/>
+    </testsuite>
+    <testsuite
+      errors="0"
+      failures="1"
+      hostname="localhost"
+      id="1"
+      name="Some section/A grouped context"
+      package="Some section/A grouped context"
+      skipped="1"
+      tests="4"
+      time="0.000031399"
+      timestamp="2023-09-12T11:08:18.360279095Z">
+        <properties/>
+        <testcase
+          classname="Some section/A grouped context"
+          file="tests/ExampleSpec.hs"
+          line="28"
+          name="throws a colourful exception"
+          time="0.000012194">
+            <failure type="error">ColourfulException</failure>
+        </testcase>
+        <testcase
+          classname="Some section/A grouped context"
+          file="tests/ExampleSpec.hs"
+          line="27"
+          name="gets skipped"
+          time="0.000007399">
+            <skipped>tests/ExampleSpec.hs:27:9
 some reason</skipped>
-    </testcase>
-    <testcase
-      file="tests/ExampleSpec.hs"
-      line="23"
-      name="also happens in a group"
-      classname="Some section/A grouped context"
-      time="0.000007421"/>
-    <testcase
-      file="tests/ExampleSpec.hs"
-      line="21"
-      name="happens in a group"
-      classname="Some section/A grouped context"
-      time="0.000006823"/>
-  </testsuite>
+        </testcase>
+        <testcase
+          classname="Some section/A grouped context"
+          file="tests/ExampleSpec.hs"
+          line="24"
+          name="also happens in a group"
+          time="0.000005839"/>
+        <testcase
```

Comment:
Are these whitespace changes expected?

I know they don't matter, but it makes it hard to verify there are no _unexpected_ differences.